src/support/timestamp.c

```diff
@@ -413,7 +413,7 @@ __time_value_validate_parent(
           __wt_time_window_to_string(tw, time_string[0]),
           __wt_time_aggregate_to_string(parent, time_string[1]));
 
-    if (tw->start_txn > parent->newest_txn)
+    if (parent->newest_txn != WT_TXN_NONE && tw->start_txn > parent->newest_txn)
         WT_TIME_VALIDATE_RET(session,
           "value time window has a start transaction after its parent's newest transaction; "
```

Comment:
Why dont we see those messages in the reproducer?

Reply:
What do you mean? I saw those messages.

Reply:
My bad, I looked at the wrong file.